build.gradle.kts

```diff
@@ -9,7 +9,7 @@ buildscript {
     }
 
     dependencies {
-        classpath("com.android.tools.build:gradle:8.5.0")
+        classpath("com.android.tools.build:gradle:8.6.1")
```

Comment:
Are gradle upgrades source of friction for apps team adoption of MSDK?

Reply:
Yeah, if they consume via source.  I noticed a warning telling me I should use 8.6 if I target API 35 though so I updated it.